README.md

```diff
@@ -268,6 +268,26 @@ editors:
   - css_select: h1 > a
 ```
 
+#### `debug_files`
+
+This editor passes its input through without modifying it.
+The content is written to a file in the given directory.
+The filename is created from the current UNIX Timestamp.
+
+This is neat when looking at steps in between editors is of interest.
+Especially for editors like [RSS](#rss) which use editors per item this can be handy to look at the steps in between.
+
+Warning: It's not recommended committing these files.
+`debug_files` should be removed before when committing the config.
+It might have unintended side effects or might spam your repository with many potentially large files.
+
+Examples:
+
+```yaml
+editors:
+  - debug_files: /tmp/website-stalker/
```

Comment:
Hm. I have a few remarks about this:

1. This creates (the first?) a side-effect for the web-stalker. Before the change all a user does is executing the stalker with a given config and a given output folder. All changes / output the stalker produced ended up in this folder. Clean and understandable. Now, the stalker also writes to "arbitrary" other folders outside what the user has supplied.
2. I am not sure `/tmp/` is the best place. It is a good place but it's not the place where I, as a user, would look first. Do you have reasons to not do `~/user_stalker_git/.debug/` or `~/user_stalker_git/tmp/`?
3. There is no clean up. In `/tmp/` this is more or less fine as it gets gc'ed from the OS but if the folder is changed or if the stalker is run on server that does not regulary gc `/tmp`, this might become an issue. (I am fine to just ignore this point, the doc is quiet clear imo)
4. The name for the debug files is `timestamp` or `timestamp.extension`, right? Imagine debugging a website that works 10 times a day but just changes its content order twice a day. Now, you would look at 12 logs and your only hint is a timestamp. This gets even worse when you simultaneous stalk & debug other sites as well. A good solution could be to leverage a directory structure: `/tmp/website-stalker/example.org/2024-01-12T13-37-43.html`. I think it is also fair to go easy and do `/tmp/website-stalker/example.org:1716797295.html` (we still have the "date created" meta from 99% of filesystems, to quickly get human-readable time)

Reply:
I thought about stuff like that too and the neat part is: it's fully customizable by the user. The user wants them in a different folder? Configure a different folder.
Two different places should write into different folders? Configure different folders for each.

Yeah, there could be some magic trying to figure out the URL and use that as a part of the directory structure, but what happens with multiple debugs, one before, one after something? It feels way cleaner to be fully customizable. After all this is a debug tool that should allow freedom.

I also thought about preventing `--commit` when `debug_files` is configured as that can mess up the repo files quickly. But that's something a user actively configured, so I think that's fine.